src/openforms/js/components/admin/form_design/variables/constants.js

```diff
@@ -91,10 +92,47 @@ const EMPTY_VARIABLE = {
   source: VARIABLE_SOURCES.userDefined,
   prefillPlugin: '',
   prefillAttribute: '',
+  prefillIdentifierRole: 'main',
   isSensitiveData: false,
   dataType: 'string',
   initial_value: '',
   serviceFetchConfiguration: null,
 };
 
-export {COMPONENT_DATATYPES, VARIABLE_SOURCES, DATATYPES_CHOICES, EMPTY_VARIABLE};
+const IDENTIFIER_ROLE_CHOICES = [
+  [
+    'main',
+    defineMessage({
+      description: 'Choices (main/authorised person) label',
+      defaultMessage: 'Main',
+    }),
+  ],
+  [
+    'authorised_person',
+    defineMessage({
+      description: 'Choices (main/authorised person) label',
+      defaultMessage: 'Authorised person',
+    }),
+  ],
+];
+
+const IDENTIFIER_ROLE_LABELS = {
```

Comment:
I don't understand the duplication here?

Can't you re-use `IDENTIFIER_ROLE_CHOICES` (using `Object.fromEntries(IDENTIFIER_ROLE_CHOICES)`) and then apply `intl.formatMessage(options[value])`, with `intl` coming from `const intl = useIntl()`